### ISSUE TITLE
PyPI release?

### ISSUE THREAD

**Issue by icemac** [zopefoundation/zc.intid#20]
It would be nice to have a new release of this package.

@jamadden @freddrake PyPI lists you as maintainers. Could you please cut a release after #19 is merged or add me (`icemac`) on PyPI, so I can cut the release myself?

**Comment by jamadden** [comment]
`icemac` now has PyPI rights :)

**Comment by icemac** [comment]
Thank you, I just released https://pypi.org/project/zc.intid/2.1.0/.
